oktoberfest/runner.py

```diff
@@ -366,11 +376,12 @@ def generate_spectral_lib(config_path: Union[str, Path]):
 
     spec_library = _speclib_from_digestion(config)
 
-    server_kwargs = {
-        "server_url": config.prediction_server,
-        "ssl": config.ssl,
+    predictor_kwargs = {
         "disable_progress_bar": True,
     }
+    predictors = {
+        model_key: pr.Predictor.from_config(config, model_key, **predictor_kwargs) for model_key in config.models
```

Comment:
since predictor_kwargs is only one, after introducing `from_config,`  you can directly provide the argument here